agentic_trader/tui.py

```
@@ -1437,47 +1871,77 @@ def _launch_service(
-    table.add_row("3", "Start orchestrator service")
-    table.add_row("4", "Request orchestrator stop")
-    table.add_row("5", "Open live monitor")
-    table.add_row("6", "Provider diagnostics")
-    table.add_row("7", "V1 readiness gates")
-    table.add_row("8", "Broker status")
-    table.add_row("9", "Back")
+    """
+    Builds a menu table of runtime control actions.
+    
+    Returns:
+        Table: A Rich Table titled for runtime control with two columns (key and action) and rows for the available runtime control options (1–9).
```

Ruff (0.15.14)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)